src/democracyworks/smartystreets/api/us_street.clj

```diff
@@ -0,0 +1,57 @@
+(ns democracyworks.smartystreets.api.us-street
+  "Client for the SmartyStreets US Street Address API
+
+  https://smartystreets.com/docs/cloud/us-street-api"
+  (:require
+   [democracyworks.smartystreets.core :as smartystreets]))
+
+(def base-url
+  "https://us-street.api.smartystreets.com/street-address")
+
+(defn parse-response
```

Comment:
For [more about `:input_index` and `:candidate_index`](https://smartystreets.com/docs/cloud/us-street-api#root).